auth/client/src/test/resources/org/wildfly/security/auth/client/test-wildfly-config-openssh-keys.xml

```diff
@@ -0,0 +1,220 @@
+dRTYzmC5iuioctKGRpzZzdavb0VGoDC40Idkr+ntbHJ38zM/2uym8JYC0PCWhtrllGyhfM
+8aX8dkQoAtY88pY9MFng4G0X9QIAzdTm2GhWASLrNlJKq25avXDJt85k105Jvw1mYu+PwH
+N95fGizYNZwCvyfY+a6R3fcVMMZ5HK3wFAVJqkUdhjqQGgykn2olIb2YE1WlF7C9aoqLTS
+MyyiCorwwRwATS+UzMFd86Xq2kfWTLAEQ/odJAOKgcgktMBVoc42QRH5MvSXsIbTLWuvpe
+pQJN9orp8Tof9UUmb1H6WOl5ArYDugh0StqPmxPy4jWkSaexLN7j16xQK62fnr+R7ydH8f
+PflhpgksXvaJouUswPPtilHDNxk0Gzpf0f63vjcSBKnezyIyTjmtUyzSyURRRVFEhiDGKQ
+xNT9ZW0OoIViI3xU2pHQ6J3guQtfbJiQ3jZlYzlVdqlhHn4oM4uk+DyO9w
+-----END OPENSSH PRIVATE KEY-----">
+                            <clear-password password="secret"/>
+                        </openssh-private-key>
+                    </key-pair>
+                </credentials>
+            </configuration>
+
+            <configuration name="test-ecdsa">
+                <credentials>
+                    <key-pair>
+                        <openssh-private-key pem="-----BEGIN OPENSSH PRIVATE KEY-----
+b3BlbnNzaC1rZXktdjEAAAAACmFlczI1Ni1jdHIAAAAGYmNyeXB0AAAAGAAAABCdRswttV
+UNQ6nKb6ojozTGAAAAEAAAAAEAAABoAAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlz
+dHAyNTYAAABBBAKxnsRT7n6qJLKoD3mFfAvcH5ZFUyTzJVW8t60pNgNaXO4q5S4qL9yCCZ
+cKyg6QtVgRuVxkUSseuR3fiubyTnkAAADQq3vrkvuSfm4n345STr/i/29FZEFUd0qD++B2
+ZoWGPKU/xzvxH7S2GxREb5oXcIYO889jY6mdZT8LZm6ZZig3rqoEAqdPyllHmEadb7hY+y
+jwcQ4Wr1ekGgVwNHCNu2in3cYXxbrYGMHc33WmdNrbGRDUzK+EEUM2cwUiM7Pkrw5s88Ff
+IWI0V+567Ob9LxxIUO/QvSbKMJGbMM4jZ1V9V2Ti/GziGJ107CBudZr/7wNwxIK86BBAEg
+hfnrhYBIaOLrtP8R+96i8iu4iZAvcIbQ==
+-----END OPENSSH PRIVATE KEY-----">
+                            <clear-password password="secret"/>
+                        </openssh-private-key>
+                    </key-pair>
+                </credentials>
+            </configuration>
+
+            <configuration name="test-ecdsa-masked-pwd">
+                <credentials>
+                    <key-pair>
+                        <openssh-private-key pem="-----BEGIN OPENSSH PRIVATE KEY-----
+b3BlbnNzaC1rZXktdjEAAAAACmFlczI1Ni1jdHIAAAAGYmNyeXB0AAAAGAAAABCdRswttV
+UNQ6nKb6ojozTGAAAAEAAAAAEAAABoAAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlz
+dHAyNTYAAABBBAKxnsRT7n6qJLKoD3mFfAvcH5ZFUyTzJVW8t60pNgNaXO4q5S4qL9yCCZ
+cKyg6QtVgRuVxkUSseuR3fiubyTnkAAADQq3vrkvuSfm4n345STr/i/29FZEFUd0qD++B2
+ZoWGPKU/xzvxH7S2GxREb5oXcIYO889jY6mdZT8LZm6ZZig3rqoEAqdPyllHmEadb7hY+y
+jwcQ4Wr1ekGgVwNHCNu2in3cYXxbrYGMHc33WmdNrbGRDUzK+EEUM2cwUiM7Pkrw5s88Ff
+IWI0V+567Ob9LxxIUO/QvSbKMJGbMM4jZ1V9V2Ti/GziGJ107CBudZr/7wNwxIK86BBAEg
+hfnrhYBIaOLrtP8R+96i8iu4iZAvcIbQ==
+-----END OPENSSH PRIVATE KEY-----">
+                            <masked-password iteration-count="100" salt="12345678" masked-password="mSGaLjrw4LM="/>
+                        </openssh-private-key>
+                    </key-pair>
+                </credentials>
+            </configuration>
+
+            <configuration name="test-ecdsa-cred-store-ref">
+                <credentials>
+                    <key-pair>
+                        <openssh-private-key pem="-----BEGIN OPENSSH PRIVATE KEY-----
```

Comment:
@ashley-abdelsayed98 Was just going through your WildFly documentation PR and wanted to check if we have tests for these cases:
- SSH authentication that make use of a key pair in PKCS format
- SSH authentication that references a KeyPairCredential from a CredentialStore

Reply:
Yes! I added those tests in my WildFly Core PR:

PKCS Test: https://github.com/ashley-abdelsayed98/wildfly-core/blob/WFCORE-4484/testsuite/manualmode/src/test/java/org/jboss/as/test/manualmode/management/persistence/RemoteSshGitRepositoryTestCase.java#L321, PKCS config: https://github.com/ashley-abdelsayed98/wildfly-core/blob/WFCORE-4484/testsuite/manualmode/src/test/resources/git-persistence/wildfly-config.xml#L85

Credential Store Reference Test: https://github.com/ashley-abdelsayed98/wildfly-core/blob/WFCORE-4484/testsuite/manualmode/src/test/java/org/jboss/as/test/manualmode/management/persistence/RemoteSshGitRepositoryTestCase.java#L411 Credential Store Config: https://github.com/ashley-abdelsayed98/wildfly-core/blob/WFCORE-4484/testsuite/manualmode/src/test/resources/git-persistence/wildfly-config.xml#L118

Reply:
Great, thanks!